sormas-backend/src/main/java/de/symeda/sormas/backend/common/StartupShutdownService.java

```diff
@@ -546,8 +546,13 @@ private void createOrUpdateDefaultUser(Set<UserRole> userRoles, String username,
 
 		User existingUser = userService.getByUserName(username);
 
+		boolean allUserRolesExist = userRoles != null && !userRoles.isEmpty() && !userRoles.stream().anyMatch(userRole -> userRole == null);
 		if (existingUser == null) {
-			if (!DataHelper.isNullOrEmpty(password)) {
+			if (!allUserRolesExist) {
+				logger.warn(
+					"User '" + firstName + " " + lastName
```

Comment:
Better write `logger.warn("User '{} {}' was not created because at least one of the user roles to be assigned cannot be found in the database.", firstName, lastName)` (here and on following log statements).